audio/voices.yaml

```diff
@@ -1,28 +1,26 @@
-AlliedEngineerVoice:
-	DefaultVariant: .wav
-	Voices:
-		Select: ienasea, ienaseb, ienasec, ienased
-		Move: ienamoa, ienamob, ienamoc
-		Action: ienaata, ienaatb, ienaatc
-		Feedback: ienafea, ienafeb, ienafec
-		Die: ienadia, ienadib, ienadic, ienadid
+EngineerVoice:
+	DefaultVariant: .wav
+	Prefixes:
```

Comment:
You can't have two infantry units with different engine names using the same shp. I tried. It didn't work, we'd need to ship sovietengineer.shp and alliedengineer.shp alongside the mod, and that'd be distributing EA assets illegally.